temboardui/plugins/monitoring/__init__.py

```diff
@@ -439,6 +439,7 @@ def collector(app, address, port, key):
                     last_pull=datetime.utcnow(),
                 )
                 worker_session.commit()
+        worker_session.close()
```

Comment:
Shouldn't we close in the non-except case as well?

Reply:
This is acheived with the one I added at the very end of the method.